### PR TITLE
Bring back the stack #28426 with Windows build fixed

### PR DESCRIPTION
**Pull request by zasdfgbnm**
@ezyang This brings back the stack https://github.com/pytorch/pytorch/pull/28426 with hopefully windows build fixed. Let's wait for the CI to see what happens.

### CODE REVIEW

File: aten/src/ATen/native/cuda/Copy.cu [hunk 1]
```diff
@@ -14,15 +14,8 @@ namespace native {
...
 
-template <typename dst_t, typename src_t>
-void copy_kernel_impl(TensorIterator& iter) {
-  gpu_kernel(iter, []GPU_LAMBDA(src_t x) -> dst_t {
-    return static_cast<dst_t>(static_cast<native::inter_copy_type_t<dst_t>>(x));
-  });
-}
-
 // device-to-device copy, does type conversion
-static void copy_device_to_device(TensorIterator& iter, bool non_blocking) {
```
Comment (zasdfgbnm) [inline aten/src/ATen/native/cuda/Copy.cu]: This line should fix the Windows build.

**Review by ezyang — APPROVED** [review]

**Comment by zasdfgbnm** [comment]
@ezyang Don't land yet. Jinkins seems to be broken and we don't know if Windows is working well.

**Comment by zasdfgbnm** [comment]
@ezyang Windows is working correctly, but there are some internal failures that I don't have access to.

**Comment by ezyang** [comment]
Not your fault. I'll try landing.

**Pull request closed without merging**